Documentation/helm-values.rst

```diff
@@ -0,0 +1,1193 @@
+     - int
+     - ``16384``
+   * - bpf.preallocateMaps
+     - Enables pre-allocation of eBPF map values. This increases memory usage but can reduce latency.
+     - bool
+     - ``false``
+   * - bpf.waitForMount
+     - Force the cilium-agent DaemonSet to wait in an initContainer until the eBPF filesystem has been mounted.
+     - bool
+     - ``false``
+   * - certgen
+     - Configure certificate generation for Hubble integration. If hubble.tls.auto.method=cronJob, these values are used for the Kubernetes CronJob which will be scheduled regularly to (re)generate any certificates not provided manually.
+     - object
+     - ``{"image":{"pullPolicy":"Always","repository":"quay.io/cilium/certgen","tag":"v0.1.4"},"podLabels":{},"ttlSecondsAfterFinished":1800}``
+   * - certgen.podLabels
+     - Labels to be added to hubble-certgen pods
+     - object
+     - ``{}``
+   * - certgen.ttlSecondsAfterFinished
+     - Seconds after which the completed job pod will be deleted
+     - int
+     - ``1800``
+   * - cleanBpfState
+     - Clean all eBPF datapath state from the initContainer of the cilium-agent DaemonSet. WARNING: Use with care!
+     - bool
+     - ``false``
+   * - cleanState
+     - Clean all local Cilium state from the initContainer of the cilium-agent DaemonSet. Implies cleanBpfState: true. WARNING: Use with care!
+     - bool
+     - ``false``
+   * - cluster.id
+     - Unique ID of the cluster. Must be unique across all connected clusters and in the range of 1 to 255. Only required for Cluster Mesh.
+     - int
+     - ``nil``
+   * - cluster.name
+     - Name of the cluster. Only required for Cluster Mesh.
+     - string
+     - ``"default"``
+   * - clustermesh.apiserver.etcd.image
+     - Clustermesh API server etcd image.
+     - object
+     - ``{"pullPolicy":"Always","repository":"quay.io/coreos/etcd","tag":"v3.4.13"}``
+   * - clustermesh.apiserver.image
+     - Clustermesh API server image.
+     - object
+     - ``{"digest":"","pullPolicy":"Always","repository":"quay.io/cilium/clustermesh-apiserver","tag":"latest","useDigest":false}``
+   * - clustermesh.apiserver.nodeSelector
+     - Node labels for pod assignment ref: https://kubernetes.io/docs/user-guide/node-selection/
+     - object
+     - ``{}``
+   * - clustermesh.apiserver.podAnnotations
+     - Annotations to be added to clustermesh-apiserver pods
+     - object
+     - ``{}``
+   * - clustermesh.apiserver.podLabels
+     - Labels to be added to clustermesh-apiserver pods
+     - object
+     - ``{}``
+   * - clustermesh.apiserver.replicas
+     - Number of replicas run for the clustermesh-apiserver deployment.
+     - int
+     - ``1``
+   * - clustermesh.apiserver.resources
+     - Resource requests and limits for the clustermesh-apiserver container of the clustermesh-apiserver deployment, such as     resources:       limits:         cpu: 1000m         memory: 1024M       requests:         cpu: 100m         memory: 64Mi
```

Comment:
I'm confused by the spacing here.

Reply:
This comes from indentation in the (YAML-formatted) comments that were used to generate this entry. I'm not sure we can bend helm-docs to generate it differently :/.